cfme/tests/test_login.py

```diff
@@ -69,6 +69,11 @@ def test_bad_password(request, appliance):
     user = User(credential=cred)
     user.name = 'Administrator'
 
-    with error.expected("Sorry, the username or password you entered is incorrect."):
+    if version.current_version <= 5.7:
```

Comment:
Use the appliance fixutre instead. `appliance.version` will give you the version. Remove the version import.